source/frontend/src/features/leases/detail/LeasePages/tenant/columns.tsx

```diff
@@ -42,12 +52,24 @@ const getColumns = (tenantTypes: SelectOption[]): ColumnWithProps<FormTenant>[]
       align: 'right',
       width: 16,
       maxWidth: 16,
-      Cell: (props: CellProps<FormTenant>) =>
-        isValidId(props.row.original.personId) ? (
-          <FaRegUser size={16} />
+      Cell: (props: CellProps<FormTenant>) => {
+        const original = props.row.original;
+        const status =
+          original.original !== undefined
```

Comment:
original.original is quite hard to read, can you clean this up a bit? Please also break this out so its not just one giant nested ternary.